frontend/src/messages/single-form/en-US.json

```diff
@@ -21,5 +21,11 @@
   "Player": {
     "globalRanking": "Global Ranking",
     "countryRanking": "Country Ranking"
+  },
+  "toast": {
```

Comment:
@apollo-dw please review

Reply:
I got tired of getting polluted build logs due to missing locale messages. so i copy-pasted English version as a temporary fix.